docs/source/transcript_policy.rst

```diff
@@ -0,0 +1,38 @@
+.. _transcript-policy:
```

Comment:
this page is mostly copy/pasted from `TranscriptSelectionPolicy.md`. Ideally I'd like to flesh it out more (is there stuff in the Variation Normalizer paper that I could reference?).

Reply:
Gave you permissions to the doc! Let me know if you'd like more info